src/main/java/games/strategy/triplea/settings/SettingsWindow.java

```diff
@@ -86,17 +86,14 @@ private static JComponent buildTab(final List<ClientSettingUiBinding> settings,
   }
 
   private static JComponent tabMainContents(final Iterable<ClientSettingUiBinding> settings) {
-    final JPanel contents = JPanelBuilder.builder()
-        .gridBagLayout()
-        .build();
-
-    final GridBagHelper grid = new GridBagHelper(contents, 3);
```

Comment:
The `GridBagHelper` import in this file is now unused.

Reply:
Thank you - I fixed it in: https://github.com/triplea-game/triplea/pull/2478/commits/1a954ce79f9e0dd348dcdf73dd34745e34783c0a#diff-ef49edd903db8a26e2b85b7973c1736eL20